spec/unit/unmanaged_files_renderer_spec.rb

```diff
@@ -97,6 +97,28 @@
     EOF
   }
 
+  let(:description_remote_dir) {
```

Comment:
Could this be created in the system description factory instead of having the json and then just call the scope?